updating/index.adoc

```diff
@@ -0,0 +1,79 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
```

Comment:
This link is not available. Could you double check if it's out of update?
> xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].

Reply:
BTW, i reviewed the doc from https://github.com/openshift/openshift-docs/blob/aa56c230747b39f3b52ae044e3e876e399e96d4c/updating/index.adoc. Is this correct page for me to review?

Reply:
Yes I believe so

Reply:
@xenolinux When I click `[Understanding {product-title} updates]` in https://github.com/openshift/openshift-docs/blob/aa56c230747b39f3b52ae044e3e876e399e96d4c/updating/index.adoc, it return me with 404.